server/controllers/usersController.js

```diff
@@ -1,9 +1,21 @@
-// CONTROLLER FOR USER MODEL
+const bcrypt = require("bcrypt");
 
-const db = require("../models/user");
+const connection = require("../dbConnection");
+const userSchema = require("../models/user");
 
-// Defining methods for the usersController
-module.exports = {
-  // define methods here
+const User = connection.model("User", userSchema);
 
+const SALT_ROUNDS = Number(process.env.SALT_ROUNDS);
+
+const create = async ({ email, password }) => {
+    const hashedPassword = await bcrypt.hash(password, SALT_ROUNDS);
+    const user = await User.create({ email, password: hashedPassword });
```

Comment:
We'll need to be using all of fields in Megan's user schema, not just email and password

Reply:
I think to make the sign up smoother I should either remove the required on the address or separate the profile info from the email and password and put it into its own profile collection.